tasks/src/tabular-classification/data.ts

```diff
@@ -0,0 +1,54 @@
+import type { TaskData } from "../Types";
+
+import { PIPELINE_DATA } from "../../../js/src/lib/interfaces/Types";
+import { TASKS_MODEL_LIBRARIES } from "../const";
+
+const taskData: TaskData = {
+	datasets: [
+		{
+			description: "Binary classification dataset based on a census on income.",
+			id:          "scikit-learn/adult-census-income",
+		},
+		{
+			description: "Multi-class dataset on iris flower species.",
+			id:          "scikit-learn/iris",
+		},
+	],
+	demo: {
```

Comment:
How will this show in the UI?

Reply:
I need to run this by @beurkinger or @gary149 😅